build_web_compilers/lib/src/dev_compiler_bootstrap.dart

```diff
@@ -193,12 +193,15 @@ String _appBootstrap({
   required String moduleScope,
   required String entrypointLibraryName,
   required String oldModuleScope,
-  required bool nativeNullAssertions,
-}) =>
-    '''
+  required bool? nativeNullAssertions,
+}) {
+  var nativeAssertsCode = nativeNullAssertions == null
```

Comment:
cc @nshahan for this part in particular, I presume it is fine to simply omit this line?

Reply:
It looks like we default to false in DDC (we still check in sound mode though) if we never call the `nativeNonNullAsserts` function, which is consistent with dart2js, so this looks good.

Reply:
Ok, that does sound like a behavior change (we defaulted it on before), but regardless it seems best to leave this up to the compilers.